core/src/lib.rs

```diff
@@ -56,4 +58,46 @@ pub type ProjectName = String32;
 
 pub type ProjectId = (ProjectName, ProjectDomain);
 
+pub type OrgId = String32;
+
+/// Org
+///
+/// A complete representation of an Org.
+/// Different from state::Org in which this type gathers
```

Comment:
You can use [doc links](https://rust-lang.github.io/rfcs/1946-intra-rustdoc-links.html)
```suggestion
/// Different from [state::Org] in which this type gathers
```